projects/organization-management/src/app/shared/user-edit-profile/user-edit-profile.component.html

```diff
@@ -0,0 +1,49 @@
+<fieldset>
+  <ish-select-title [form]="form" controlName="title" [titles]="titles"></ish-select-title>
+  <ish-input
+    [form]="form"
+    controlName="firstName"
+    label="account.address.firstname.label"
+    [errorMessages]="{
+      required: 'account.user.new.firstname.error.required'
+    }"
+  ></ish-input>
+  <ish-input
+    [form]="form"
+    controlName="lastName"
+    label="account.address.lastname.label"
+    [errorMessages]="{
+      required: 'account.user.new.lastname.error.required'
+    }"
+  ></ish-input>
+</fieldset>
+<fieldset *ngIf="!editing">
+  <ish-input
+    [form]="form"
+    controlName="login"
+    label="account.user.new.login.label"
+    [errorMessages]="{
+      required: 'account.user.new.login.error.required',
+      email: 'account.user.new.login.error.email'
+    }"
+  ></ish-input>
+  <ish-input
+    [form]="form"
+    controlName="email"
+    label="account.user.email.label"
+    [errorMessages]="{
+      required: 'account.update_email.email.error.notempty',
+      email: 'account.update_email.email.error.email'
+    }"
+  ></ish-input>
+</fieldset>
+<fieldset>
+  <ish-input [form]="form" controlName="department" label="account.profile.department.label"></ish-input>
```

Comment:
some localizations keys match the profile ones, should we re-use them or create new ones?